lib/tasks/data/orphaned_records_queries.sql

```diff
@@ -0,0 +1,43 @@
+SELECT COUNT(*) FROM spree_stock_movements LEFT JOIN spree_stock_items ON spree_stock_movements.stock_item_id = spree_stock_items.id WHERE spree_stock_items.id IS NULL
+
+SELECT COUNT(*) FROM spree_stock_locations LEFT JOIN spree_states ON spree_stock_locations.state_id = spree_states.id WHERE spree_states.id IS NULL
+
+SELECT COUNT(*) FROM spree_stock_locations LEFT JOIN spree_countries ON spree_stock_locations.country_id = spree_countries.id WHERE spree_countries.id IS NULL
```

Comment:
This results in 1 on every server. The stock locations don't have a state or country. So if `state_id` is null then we shouldn't count it.